source/module_ri/Exx_LRI_interface.h

```diff
@@ -19,15 +19,24 @@ template<typename T, typename Tdata>
 class Exx_LRI_Interface
 {
 public:
+    using TC = std::array<int, 3>;
+    using TAC = std::pair<int, TC>;
+
     /// @brief  Constructor for Exx_LRI_Interface
     /// @param exx_ptr
     Exx_LRI_Interface(std::shared_ptr<Exx_LRI<Tdata>> exx_ptr) : exx_ptr(exx_ptr) {}
     Exx_LRI_Interface() = delete;
 
     void write_Hexxs(const std::string &file_name) const;
     void read_Hexxs(const std::string& file_name);
-    
-    using TAC = std::pair<int, std::array<int, 3>>;
+
+    void write_Hexxs(const std::string& file_name, const UnitCell& ucell) const;
+    void read_Hexxs(const std::string& file_name, const UnitCell& ucell);
+
+    std::map<Abfs::Vector3_Order<int>, std::map<size_t, std::map<size_t, Tdata>>>
```

Comment:
There is a function with complex return, you'd better add annotations to describe the function and the returned value.